launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml

```diff
@@ -229,8 +230,9 @@
     <let name="merger/output/objects" value="$(var output/objects)" unless="$(var use_object_filter)"/>
     <include file="$(find-pkg-share object_merger)/launch/object_association_merger.launch.xml">
       <arg name="input/object0" value="temporary_merged_objects"/>
-      <arg name="input/object1" value="clustering/objects"/>
+      <arg name="input/object1" value="detection_by_tracker/objects"/>
       <arg name="output/object" value="$(var merger/output/objects)"/>
+      <arg name="priority_mode" value="1"/>
```

Comment:
detection by tracker objects are prioritized only in camera_lidar_fusion.
Isn't is necessary to change also in lidar_based_detection.launch.xml?

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/1683#discussion_r972608374
I was making a mistake. I'll change to confidence based mode.

Reply:
fixed at https://github.com/autowarefoundation/autoware.universe/pull/1683/commits/c7fe9e613c864d42cb8cb019e36c9aa8ee7fc6de